fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -68686,6 +68681,91 @@ relationship: overlaps FBbt:00007053 ! lateral horn
 
+[Term]
+id: FBbt:00047216
+name: larval DMS cell of pars intercerebralis
+def: "Neuron of the larval pars intercerebralis (PI) that expresses dromyosuppressin (DMS) as well as dimmed. There are two of these cells per hemisphere. These cells are distinct from the PI neurons that express insulin like peptides, diuretic hormone 44 and SIFamide." [FlyBase:FBrf0210278]
+synonym: "PI 13-14" EXACT [FlyBase:FBrf0210278]
+relationship: capable_of GO:0030072 ! peptide hormone secretion
+relationship: expresses http://flybase.org/reports/FBgn0011581
+relationship: has_soma_location FBbt:00111670 ! larval pars intercerebralis
+created_by: CP
+creation_date: 2018-06-05T15:13:30Z
+
+[Term]
+id: FBbt:00047217
+name: larval SIFamide cell of pars intercerebralis
+def: "Neuron of the larval pars intercerebralis (PI) that expresses SIFamide (SIFa) as well as dimmed. There are two of these cells per hemisphere. These cells are distinct from the PI neurons that express insulin like peptides, diuretic hormone 44 and dromyosuppressin." [FlyBase:FBrf0210278]
+synonym: "PI 8-9" EXACT [FlyBase:FBrf0210278]
+relationship: capable_of GO:0030072 ! peptide hormone secretion
+relationship: expresses http://flybase.org/reports/FBgn0053527
+relationship: has_soma_location FBbt:00111670 ! larval pars intercerebralis
+created_by: CP
+creation_date: 2018-06-06T08:14:14Z
+
+[Term]
+id: FBbt:00047243
+name: dorsal ventral muscle
+def: "An indirect flight muscle that functions as a wing elevator. It extends dorsoventrally, connecting the dorsal region of the mesothoracic tergum and the ventrolateral margin of the mesothoracic phragma (Williams and Williams, 1943). These vertical muscles function to elevate the wings by drawing down on the scutum (Miller, 1950)." [FlyBase:FBrf0006004, FlyBase:FBrf0007735, FlyBase:FBrf0064793]
+synonym: "DVM" EXACT [FlyBase:FBrf0064793]
+is_a: FBbt:00003361 ! indirect flight muscle
+created_by: CP
+creation_date: 2018-06-14T08:04:07Z
+
+[Term]
+id: FBbt:00047244
+name: external muscle of fourth axillary hg1
+def: "Direct flight muscle that inserts into the fourth axillary (posterior notal wing process) and functions in carrying out lateral tip movements (Wisser and Nachtigall, 1984). It shares a tendon with the external muscle of fourth axillary hg4 (Williams and Williams, 1943). In contrast to most other wing muscles, hg1 is larger in males than in females, which is thought to allow for louder sine song (Shirangi et al., 2013). It is one of only two flight muscles that express doublesex (Shirangi et al., 2013)." [doi:10.1007/BF00312139, FlyBase:FBrf0006004, FlyBase:FBrf0098527, FlyBase:FBrf0223321]
+comment: This muscle is not illustrated in Miller 1950 (FBrf0007735).
+synonym: "hg1 muscle" EXACT []
+synonym: "posterior notal wing process muscle hg1" EXACT [FlyBase:FBrf0098527]
+is_a: FBbt:00003387 ! axillary muscle
+relationship: expresses http://flybase.org/reports/FBgn0000504
+relationship: synapsed_by FBbt:00047246 ! hg1 motor neuron
```

Comment:
Please add 
subClassOf: 'capable of' some 'male courtship behavior, veined wing generated song production' (GO:0045433)
Ditto hg4 & hg1.

Reply:
not capable_of_part_of? (if neurons and muscles are each doing part)